packages/suite/src/views/onboarding/steps/SelectBackupType.tsx

```diff
@@ -130,13 +132,24 @@ const OptionGroupHeading = styled.div`
     align-items: center;
 `;
 
-const OptionStyled = styled.div`
+const OptionStyled = styled.div<{ $hasHoverInteraction?: boolean }>`
     display: flex;
     flex-direction: row;
 
     gap: ${spacingsPx.md};
     align-items: center;
     cursor: pointer;
+
+    ${({ $hasHoverInteraction }) =>
+        $hasHoverInteraction === true
+            ? css`
+                  &:hover {
+                      background-color: ${({ theme }) => theme.backgroundSurfaceElevation2};
+                      margin: -${spacingsPx.sm};
```

Comment:
temporary hack just to see the UI, maybe we can figure out more clear way @jvaclavik ?

Reply:
Not great, but totally understand the use case.